tests/projectscarouse.tests.spec.ts

```diff
@@ -0,0 +1,90 @@
+describe('Home Page/Project - Check the carousel of the images', ()=>{
+    it('test image carousel functionality', async ()=>{
```

Comment:
please add almost one except to validate each section to ensure that functionality is correct